src/microbe_stage/MicrobeStage.tscn

```diff
@@ -2252,6 +2252,7 @@ custom_styles/separator = SubResource( 36 )
 anims/FadeInOut = SubResource( 37 )
 
 [node name="PauseMenu" parent="." instance=ExtResource( 11 )]
+visible = true
```

Comment:
Godot _really_ insist for this to be visible, last time I got this the actual menu is not shown and this value is basically just saved here for no reason. Could be wrong in this case but I guess we can just let this be?

Reply:
Good catch. I'll fix this in a bit.

Reply:
Looks like truly this change could not be reverted with the Godot editor so I had to manually do this...

Reply:
I know right.